src/main/java/com/reckue/post/services/realizations/CommentServiceRealization.java

```diff
@@ -35,12 +34,7 @@
      */
     @Override
     public Comment create(Comment comment) {
-        comment.setId(Generator.id());
-        if (!commentRepository.existsById(comment.getId())) {
-            return commentRepository.save(comment);
-        } else {
-            throw new ModelAlreadyExistsException("Comment already exists");
-        }
+        return commentRepository.save(comment);
     }
```

Comment:
As well as you need to delete in classes of package "controllers.apis" the annotation to method create -  "@ApiResponse(code = 409... " as unused.